addons/point_of_sale/static/src/js/Screens/PaymentScreen/PaymentScreen.js

```diff
@@ -81,6 +81,9 @@ odoo.define('point_of_sale.PaymentScreen', function (require) {
         }
         addNewPaymentLine({ detail: paymentMethod }) {
             // original function: click_paymentmethods
+            if(!this.env.pos.get_order().check_paymentlines_rounding()) {
```

Comment:
If we do this we are not checking the paymentline that is being added right after. And only checking it when we do the validateOrder. When it was checked in more places before the fix. If we want to keep it clean for the notification we can create a function that will show the errorPopup and not modify the places where we check the paymentlines rounding.

Reply:
It doesn't seem to break anything because no test is failing. But modifying the is_paid method seems like a bad idea to me